packages/fuels-accounts/src/provider/supported_versions.rs

```diff
@@ -1,7 +1,7 @@
 use semver::Version;
 
 fn get_supported_fuel_core_version() -> Version {
-    "0.21.0-rc.1".parse().unwrap()
```

Comment:
Nice catch -- how can we make sure this value stays up-to-date? I mean, is there a way to automate that in our CI? I think it will be tricky to remember to update this constantly.

Reply:
I think @Br1ght0ne has something in mind over on #1190 . 

I'll review it today. Something caught my eye that might be a problem.

Reply:
#1190 should be good to go!